components/number/template.rst

```diff
@@ -34,7 +34,8 @@ Configuration variables:
 - **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
   number. Defaults to ``60s``.
 - **optimistic** (*Optional*, boolean): Whether to operate in optimistic mode - when in this mode,
-  any command sent to the template number will immediately update the reported state.
+  any command sent to the template number will immediately update the reported state. 
+  This must be set to ``true`` inorder for changes from Home Assistant to be applied.
```

Comment:
This is not true, you can also have a `set_action` that does something based on the changed value from HA and a `lambda` that reports back what the current value is.